documentation/src/main/asciidoc/topics/ref_cache_store_jdbc.adoc

```diff
@@ -0,0 +1,25 @@
+[id='jdbc_cache_store']
+= JDBC String-Based Cache Stores
+JDBC String-Based cache stores, `JdbcStringBasedStore`, use JDBC drivers to load and store values in the underlying database.
+
+`JdbcStringBasedStore` stores each entry in its own row in the table, instead
+of grouping multiple entries into each row, to increase throughput for
+concurrent loads. `JdbcStringBasedStore` also uses a pluggable bijection that
```

Comment:
I think we can remove the "instead of grouping ..." comment. This was relevant in the past, because we had an alternative JDBC store implementation that did group multiple entries per row, but as that is no longer the case we can remove this detail.